content/docs/0.6.0/usecases/quality-gates/index.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Keptn Quality Gates only
+description: Describes how Keptn allows to use quality gates without deployment and testing features of Keptn.
+weight: 26
+keywords: [quality-gates]
+aliases:
+---
+
+Describes how Keptn allows to use quality gates without deployment and testing features of Keptn.
+
+## About this tutorial
+
+Let's say you want to use your existing tools for deploying and testing your applications - you can still use Keptn Quality Gates for extensive evaluation of Service Level Objectives (SLOs).
+
+*A brief recap of SLO and SLI:* A service level objective (SLO) is a target value or range of values for a service level that is measured by a service level indicator (SLI). An SLI is a carefully defined quantitative measure of some aspect of the level of service that is provided. By default, the following SLIs can be used for evaluation, inspired by the [Site Reliability Engineering](https://landing.google.com/sre/sre-book/chapters/service-level-objectives) book from Google:
+
+* *Response Time*: The time it takes for a service to execute and complete a task or how long it takes to return a response to a request.
+* *System Throughput*: The number of requests per second that have been processed.
+* *Error Rate*: The fraction of all received requests that produced an error.
+
+For more information about SLO and SLI, please take a look at [Specifications for Site Reliability Engineering with Keptn](https://github.com/keptn/spec/blob/0.1.1/sre.md).
+
+## Prerequisites
+
+* Running Keptn installation or a quality gates only installation as explained below
```

Comment:
Correct me if I am wrong: An already existing keptn installation might not work for this use-case as the quality-gates standalone removes distributors, right?

I'd like to avoid people running into issues with this, so maybe we actually require the Keptn Qualitygates Standalone installation for this use case using `keptn install ... --use-case=quality-gates`